modules/gateway/peers_test.go

```diff
@@ -653,6 +653,8 @@ func TestDisconnect(t *testing.T) {
 		if err != nil {
 			t.Fatal("accept failed:", err)
```

Comment:
fixed all of the offenders in the gateway, not sure about the other packages though (commit incoming).